assets/master.js

```diff
@@ -1,6 +1,6 @@
-(function() {
-  'use strict';
+'use strict';
```

Comment:
`strict` rule from `eslint` complaints - do you want me to add `/* eslint-disable */` instead?